lib/workload/stateless/stacks/filemanager/docs/API_GUIDE.md

```diff
@@ -139,6 +139,27 @@ curl --get -H "Authorization: Bearer $TOKEN" --data-urlencode "attributes[portal
 "https://file.dev.umccr.org/api/v1/s3" | jq
 ```
 
+## Multiple keys
+
+The API supports querying using multiple keys with the same name. This represents an `or` condition in the SQL query, where
+records are fetched if any of the keys match. For example, the following finds records where the bucket is either `bucket1`
+or `bucket2`:
+
+```sh
+curl -H "Authorization: Bearer $TOKEN" "https://file.dev.umccr.org/api/v1/s3?bucket[]=bucket1&bucket[]=bucket2" | jq
+```
+
+Note that the extra `[]` is required in the query parameters to specify multiple keys with the same name. Multiple keys
```

Comment:
What is the reason behind it where it needs to have the `[]` for a multiple key? Will this be mutually exclusive if there is a params that doesn't have `[]`? Say `...s3?bucket=bucket1&bucket[]=bucket2`

Reply:
It's required in the library that parses the query string: https://github.com/samscott89/serde_qs. I think the choice by the author has to do with matching the style of ruby on rails. 

Yes, this is mutually exclusive, you can't combine keys with `[]` and without it.

For example, this works:
1. `/api/v1/s3?key=...`
2. `/api/v1/s3?key[]=...`
3. `/api/v1/s3?key[]=...&key[]=...`

But this results in an error:
1. `/api/v1/s3?key=...&key[]=...`